<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editing main database</string>
  </property>
  <widget class="QTableView" name="tableViewMain">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>670</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>491</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="b_submit">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_revert">
      <property name="text">
       <string>&amp;Revert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_contact_inf">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>311</y>
     <width>660</width>
     <height>170</height>
    </rect>
   </property>
   <property name="title">
    <string>Contact information</string>
   </property>
   <widget class="QLineEdit" name="le_fullname_contact">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>20</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Full name contact</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>120</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_phone_contact">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>48</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>120</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_birthday_contact">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>78</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>120</width>
      <height>14</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>e-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email_contact">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>108</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>141</y>
      <width>120</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Google ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_id_google">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>138</y>
      <width>510</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
